### PR TITLE
APIGOV-27684 add metrics filtering fields

### PR DESCRIPTION
**Pull request by alrosca**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Updated Apigee X documentation to reflect the Apigee TA metrics filtering 

## Deploy preview link

http://pr-432.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_apigee_x/

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 2de2ff0 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-432.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md [hunk 1]
```diff
@@ -141,6 +141,8 @@ The installation procedure will prompt for the following:
    * **Project ID**: the Project ID for your Google Cloud Platform project
    * **Developer Email**: the email address of a developer, defined in Apigee, that will be given ownership of all Applications
    * **Client Email**: the email address, principal name, for the service account in GCP that has the role to discovery Apigee resources
+   * **Filter Metrics**: this field determines if api metrics filtering is wanted. Defaults to true
+   * **Filtered APIs**: list that should contain apis for which metrics are wanted. Leave empty to use all the discovered apis instead
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_apigee_x/_index.md:145]: Maybe give an example of the format of the list for empty and a list containing API matching with/without wildcards.

**Review by jcollins-axway — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_apigee_x/_index.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/connect_apigee_x/_index.md:145]: Thank you for adding the examples

**Review by robertengaxway — APPROVED** [review]
Approved

**Review by sbolosan — APPROVED** [review]
Content is good. Is there a better word than "wanted"?

**Review by vivekschauhan — APPROVED** [review]

**Pull request merged**